curve25519-dalek/src/backend/mod.rs

```diff
@@ -46,14 +46,14 @@ pub mod vector;
 enum BackendKind {
     #[cfg(curve25519_dalek_backend = "simd")]
     Avx2,
-    #[cfg(all(curve25519_dalek_backend = "simd", nightly))]
+    #[cfg(all(curve25519_dalek_backend = "unstable_avx512", nightly))]
```

Comment:
If both `backend="simd"` and `backend="unstable_avx512"` are set, then are both variants included?

Reply:
yes. unstable_avx512 includes simd as fallback given it would get selected as default in any case with simd capable platform.